install/pre-installation.md

```diff
@@ -563,6 +563,19 @@ successfully.
    (for example, editing configuration or CSI inputs), then restart from the beginning of the
    [Initialize the LiveCD](Create_System_Configuration_Using_SHCD.md#6-initialize-the-livecd) procedure.
 
+   The following test may fail if the management network switches have not been configured.
+   This may be the case if this is the first time the system is being installed with CSM. Configuring switches is covered in the next topic.
```

Comment:
Should we indicate that if the failure occurs, it can be ignored (because configuring the switches is next)?

Reply:
Sounds good. I've reworded the text. Tell me what you think.

Reply:
Looks great!